client/src/components/EmailSignup.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class='section form-wrapper columns'>
+    <div class='header-section column is-half is-centered'>
+      <div class='h1-header-large centered-text'>
+        {{ MESSAGES.SUBSCRIBE_HEADER }}
+      </div>
+      <div class='h2-header-large centered-text mt-4'>
+        {{ MESSAGES.SUBSCRIBE_SUBHEADER }}
+      </div>
+    </div>
+    <div id='hubspotForm' v-once></div>
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import { LandingPageMessages } from '../assets/messages/landing';
+
+/**
+ * Section containing form to sign up for LeadOut data updates.
+ *
+ * This wraps a hubspot form which is configured and styled in the hubspot editor.
+ */
+export default defineComponent({
+  name: 'EmailSignup',
+  data() {
+    return {
+      MESSAGES: LandingPageMessages,
```

Comment:
nit: for consistency, camelcase the data vars

Reply:
done!